src/codegen/sys/statics.rs

```diff
@@ -0,0 +1,362 @@
+use std::io::{Result, Write};
+
+pub fn generate<W: Write>(w: &mut W) -> Result<()>{
```

Comment:
No this file from this branch, for compiling.
File will greatly reduced when generated structs for Class and Interface, but still last here.